website/tests/pages/user/group/index.spec.ts

```diff
@@ -25,4 +26,49 @@ test.describe('The group page', () => {
 
         await groupPage.verifyUserIsNotPresent(testUser);
     });
+
+    test('should edit group info and see changes afterwards', async ({ groupPage, loginAsTestUser }) => {
+        await loginAsTestUser();
+
+        await groupPage.goToUserPage();
+        await groupPage.goToGroupCreationPage();
+
+        const groupName = v4();
+        await groupPage.createGroup(groupName);
+        const newName = v4();
+        const newInstitution = v4();
+        const newEmail = `${v4()}@example.com`;
+        const newCountryIndex = 2;
+        const newCountry = listOfCountries[newCountryIndex];
```

Comment:
```suggestion
        const newCountry = listOfCountries[newCountryIndex-1];
```
Seems to be off-by-one